wg-lts/charter.md

```diff
@@ -0,0 +1,81 @@
+* what do users/vendors do with EOL clusters today? what would they do if EOL was extended by N months/years?
+
+With this information, we will investigate changes the Kubernetes project could make to address these needs. A non-exhaustive set of changes to investigate include:
+* lengthening support period for all minor versions
+* lengthening support period for specific minor versions
+* applying security fixes to release branches past current EOL without cutting additional patch releases
+* applying security and "critical" fixes to release branches past current EOL without cutting additional patch releases
+* expanding supported skew
+* improving supported upgrade patterns for clusters at EOL
+
+### In Scope
+
+- Collecting input to better define long term support with regard to Kubernetes OSS releases
+- Creating and prioritizing a list of areas that require investments to improve long term supportability
+- Initiate and drive cross-SIG changes related to long term support
+
+### Out of scope
+
+- Designing and executing on changes clearly falling into individual SIG
+  responsibilities.
+
+## Special Powers
+
+None
+
+## Stakeholders
+
+Any changes identified by this working group will require involvement and investment from the following SIGS:
+
+- SIG Architecture
+  High-level input on requirements.
+- SIG Cluster Lifecycle
+  Input on cluster upgrade mechanics.
+- SIG K8s Infra
+  Input on infrastructure costs related to increased support period(s).
+- SIG Release
+  Input on maintaining older branches and additional releases.
+- SIG Testing
+  Input on testing impacts related to upgrades and longer test periods. 
+  
+Improvements identified will likely involve every SIG, but the list above are identified as the primary stakeholders.
+
+## Deliverables
+
+The artifacts the group is supposed to deliver include:
+- Survey results better describing Kubernetes version use patterns, deployment constraints, and upgrade patterns.
+- Recommend changes Kubernetes SIGS can make that will provide broad benefits in a sustainable/affordable way. This would likely take the form of one or more KEPs.
+- Recommend ways users/vendors who want to maximize Kubernetes support can consume Kubernetes
+
+Any changes identified will be owned by corresponding SIGs.
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in
+[sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Timelines and Disbanding
+
+The exact timeline for this working group is hard to define at this time. If we are unable to define a common long term support definition for Kubernetes, we will disband the working group. If we are unable to define improvements related to an agreed upon definition, we will also disband the working group.
```

Comment:
Should we write periodic updates and reviews to sig-arch/steering/someone else in here? Perhaps a 6-monthly update, and a review after 12 months?

Feels like having some deadlines to produce something might help keep the discussion focussed.

Reply:
@youngnick  that's a good suggestion, I will incorporate your feedback.